app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
```

Comment:
You shouldn't need multiple CommentsController classes. You should be able to mange with one using the params hash or format of the url to figure out what the commentable is for the current request.